### PR TITLE
GitHub - Add and modify commands

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added the ***Github-list-files*** command.
- Added repository and organization arguments to ***GitHub-list-pr-files*** command.
- Added repository and organization arguments to ***github-get-pull-request*** command.
- Added ***GitHub.Issue. Organization*** output to the commands: ***GitHub-create-issue, GitHub-close-issue, GitHub-update-issue, GitHub-list-all-issues, GitHub-search-issues***

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GitHub/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "GitHub",
     "description": "Manage GitHub issues and pull requests directly from Cortex XSOAR",
     "support": "xsoar",
-    "currentVersion": "1.1.11",
+    "currentVersion": "1.1.12",
```
Comment (yaakovi) [inline Packs/GitHub/pack_metadata.json:5]: this is an enhancement, bump the version by minor
```suggestion
    "currentVersion": "1.2.0",
```

**Review by yaakovi — COMMENTED** [review]
nice work

**Comment by ShirleyDenkberg** [comment]
@yaakovi @adi88d Doc review completed.  As we discussed, Adi will update the output in the README (under the Issues heading, the Owner field should be changed to Organization. This occurs is several lines in the README.

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 2]
```diff
@@ -117,6 +117,9 @@ script:
     - contextPath: GitHub.Issue.Closed_by
       description: User who closed the issue.
       type: String
+    - contextPath: GitHub.Issue.Organization
+      description: The repository owner.
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:121]: update in all places (yml and readme)
```suggestion
      description: The repository organization.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**